test/test_model.py

```diff
@@ -103,7 +107,8 @@ def check_waveform(assertion, generated_waveform):
 
     # Expect most of the power to be at the 3 frequencies we trained
     # on.
-    assertion(expected_power, 0.9 * power_sum)
+    assertion(expected_power, 0.75 * power_sum)
```

Comment:
OK I finally get your point now about the "training data not passing the test." I'll have to think about that. I remember when I first wrote it I looked at the power spectrum of the training data and there were three very sharp peaks at the three frequencies; hard to see how it couldn't pass. Maybe bleeding into adjacent bins increased when we decreased the sample rate of the test to 2 KHz (to make it run faster). Not sure.